src/Core/src/Platform/Standard/ViewExtensions.cs

```diff
@@ -2,6 +2,8 @@ namespace Microsoft.Maui.Platform
 {
 	public static partial class ViewExtensions
 	{
+		public static void Initialize(this object nativeView, IView view) { }
```

Comment:
If this extension method is going to only make sense on Android, why not create an specific class under the Platform/Android folder and use compilation directives to invoke it?. In that way we avoid to have some empty public extensions methods (Windows, iOS).